editor/src/components/canvas/canvas-strategies/strategies/absolute-resize-bounding-box-strategy.tsx

```diff
@@ -183,9 +183,10 @@ export function absoluteResizeBoundingBoxStrategy(
                 return []
               }
 
-              const elementIsGroup = treatElementAsGroupLike(
+              const elementIsGroup = allowGroupTrueUp(
```

Comment:
here too, I'd make it more relaxed and use treatElementAsGroupLike

Reply:
Fixed in c381620da.